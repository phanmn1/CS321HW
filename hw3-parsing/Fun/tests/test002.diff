1,13c1
< Function
<   int
<   "foo"
<   Formal
<     int
<     "num"
<   Formal
<     boolean
<     "hah"
<   Block
<     Empty
<     Print
<       Id("hah")
---
> Invalid syntax at (13,2), num
