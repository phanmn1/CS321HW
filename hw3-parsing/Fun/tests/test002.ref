Function
  int
  "foo"
  Formal
    int
    "num"
  Formal
    boolean
    "hah"
  Block
    Empty
    Print
      Id("hah")
