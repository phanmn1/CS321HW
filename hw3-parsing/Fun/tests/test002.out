Invalid syntax at (13,2), num
